### DISCUSSION TITLE
Similar records clustering

### DISCUSSION THREAD

**Discussion by kodeplex** [moj-analytical-services/splink#1265] [General]
I am quite new and would like to know if Splink clusters the result of similar records, I mean if it groups them in a group number so that it's eaasy to work after.

For example:
---Cluster Number--Company Name--
    291 Microsoft Corp
    291 Microsoft Corporation
    
   578 Oracle Corp
   578 Oracle Corporation
   
   I have tried to research this topic already but failed to understand properly if other's have already asked what I am looking for.

**Comment by RobinL** [comment]
Yes, the [introductory tutorial](https://moj-analytical-services.github.io/splink/demos/00_Tutorial_Introduction.html) covers clustering.  The relevant section is here:

https://moj-analytical-services.github.io/splink/demos/05_Predicting_results.html#clustering
